package.json

```diff
@@ -34,7 +34,7 @@
     "redux": "^3.7.2",
     "redux-persist": "^4.6.0",
     "redux-thunk": "^2.3.0",
-    "spdx-expression-parse": "^3.0.0",
+    "spdx": "github:clearlydefined/spdx",
```

Comment:
you can update this to `"@clearlydefined/spdx": "0.1.0"`
https://www.npmjs.com/package/@clearlydefined/spdx

Reply:
great! will do.